proto/proto/ibc.proto

```diff
@@ -38,6 +39,31 @@ message IBCAction {
   }
 }
 
+message ICS20Withdrawal {
+  // the chain ID of the destination chain for this ICS20 transfer
+  string destination_chain_id = 1;
+
+  // a transparent value consisting of an amount and an asset ID.
+  crypto.Value value = 2;
+
+  // the address on the destination chain to send the transfer to 
+  string destination_chain_address = 3;
+
+  // a "sender" penumbra address to use to return funds from this withdrawal.
+  // this should be an ephemeral address
+  crypto.Address sender = 5;
+
+  // the height (on Penumbra) at which this transfer expires (and funds are sent
+  // back to the sender address?). NOTE: if funds are sent back to the sender,
+  // we MUST verify a nonexistence proof before accepting the timeout, to
+  // prevent relayer censorship attacks. The core IBC implementation does this
+  // in its handling of validation of timeouts.
+  .ibc.core.client.v1.Height timeout_height = 6;
```

Comment:
Elsewhere we use `uint64` heights, is there a reason to use the IBC proto type in our transaction action?

Reply:
IBC heights are slightly different- they contain both an epoch and a height. We don't have epochs so the epoch is always 0. Using an IBC height makes it easier to interoperate with other chains, but if this is only going to be used for penumbra heights then we can just use a uint64